PhysicsTools/Heppy/src/TriggerBitChecker.cc

```diff
@@ -53,9 +53,9 @@ namespace heppy {
     return outcome;  // true only if all paths are unprescaled
   }
 
-  int TriggerBitChecker::getprescale(const edm::EventBase &event,
-                                     const edm::TriggerResults &result_tr,
-                                     const pat::PackedTriggerPrescales &result) const {
+  double TriggerBitChecker::getprescale(const edm::EventBase &event,
```

Comment:
do we wish to consider protecting this against people containing to call this expecting an int via the template with default to int and static assert for having an double/FracPrescale